src/custom-element.js

```diff
@@ -1194,6 +1209,9 @@ function createBaseCustomElementClass(win) {
      */
     viewportCallback(inViewport) {
       assertNotTemplate(this);
+      if (inViewport) {
```

Comment:
Do we want to do this via `viewportCallback`? Or InOb? With viewportCallback we kind of testing against the same tracking system as `layoutCallback` and we could compound errors.